src/virtac/atip_server.py

```diff
@@ -106,13 +106,17 @@ def __init__(
             self._create_feedback_records(feedback_csv, disable_emittance)
         if mirror_csv is not None:
             self._create_mirror_records(mirror_csv)
-        print(f"Finished creating all {len(self.all_record_names)} records.")
+        self._record_names = self._get_all_record_names()
+        print(f"Finished creating all {len(self.record_names)} records.")
 
     @property
-    def all_record_names(self):
-        """A dictionary with all the names of records created by this server
-        as the keys and the corresponding record (and mirror) objects as the
-        values.
+    def record_names(self):
+        return self._record_names
+
+    def _get_all_record_names(self):
```

Comment:
If this function is only used to update `self._record_names`, why not rename it to _update_record_names() and assign `self._record_names `directly?

I think it makes things clearer and reduces the line count.